__tests__/__system__/api/methods/start/Start.region.system.test.ts

```diff
@@ -106,11 +106,6 @@ describe("IMS start region", () => {
         expect(error).toBeTruthy();
         expect(response).toBeFalsy();
         expect(error.mDetails.msg).toContain("IMS member name is required");
-    });
-
-    it("should fail to start region due to invalid membername value", async () => {
```

Comment:
Should this test be renamed rather than merged with another one? Keeping the number of `expect` calls per test minimal can make it easier to debug tests when they fail, but I'm probably being too picky here 😢

Reply:
Very good point though.
it's always good to keep the tests as small as possible 😋